src/tasks/prettier-format.ts

```diff
@@ -1,5 +1,6 @@
 import { execa } from "execa";
 
+// TODO: Instead of using execa, use prettier pacakge from cli to format targetDir
```

Comment:
package*

Reply:
Ohh tysm !! lol for some reason even github copilot has started making spell mistake like me :(